docs/spec.md

```diff
@@ -2237,7 +2237,7 @@ contain the sizes of the slice for each dimension.
 
 More formally, `result[i0, ..., iR-1] = operand[j0, ..., jR-1]` where:
 
-* `jd = adjusted_start_indices[d][] + id`.
+* `jd = adjusted_start_indices[d] + id`.
```

Comment:
`id = adjusted_start_indices[d] + jd` like below?

Reply:
I also doubted the same before, but fortunately this formulation is correct. So we should not need this.

Reply:
The way I look at it is: 
index of larger window (the operand in this case) = start indices + the indices of the slice (the result here)

Reply:
Please let me know if this makes sense.